packages/dds/legacy-dds/src/index.ts

```diff
@@ -3,11 +3,12 @@
  * Licensed under the MIT License.
  */
 
+/* eslint-disable import/no-internal-modules */
```

Comment:
you shouldn't disable this, you should add an index.ts in the signal directory and add all the export there, and then export from the index here.

Reply:
this is a common pattern in a repo, so you should be able to find examples if you need them